spec/controllers/reports_controller_spec.rb

```diff
@@ -2,20 +2,21 @@
 
 describe ReportsController, type: :controller do
   before(:all) do
```

Comment:
I think it should be fine so long as the spec is cleaned up (there's an `after(:all)` here), but I could mock these as well.